CryptoAnalysis/src/main/java/crypto/reporting/JSONReporter.java

```diff
@@ -113,16 +117,15 @@ public void afterAnalysis() {
 		summary.put(JSONConfig.VIOLATIONS, violations);
 		json.put(JSONConfig.SUMMARY, summary);
 	
-		if (outputFolder != null) {
 		try {
 			 ObjectMapper mapper = new ObjectMapper();
 			 ObjectWriter writer = mapper.writer(new DefaultPrettyPrinter());
 			 writer.writeValue(Paths.get(outputFolder + File.separator+"CogniCrypt-JSON-Report.json").toFile(), json);
+			 LOG.info("JSON Report generated to file : "+ outputFolder + File.separator+"CogniCrypt-JSON-Report.json");
 	     } 
 		catch (IOException e) {
 	    	 e.printStackTrace();

```

Comment:
Use LOG.error here instead

Reply:
It is not an error. It is just a message to display the report location in CLI.

Reply:
Sorry.. The suggested change was for `e.printStackTrace();`